.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: go
+sudo: false
+go:
+  - "1.11.5"
+before_install:
+  - go get golang.org/x/lint/golint
```

Comment:
What do you think about doing it in the Makefile instead? I don't know how go people typically do it, but [the way I did it in terraform](    https://github.com/openshift-metalkube/terraform-provider-ironic/blob/master/Makefile#L18-L22) works locally.

Reply:
Sure, that's reasonable.